jwql/website/apps/jwql/data_containers.py

```diff
@@ -65,6 +68,36 @@
 REPO_DIR = os.path.split(PACKAGE_DIR)[0]
 
 
+def get_table_view_components(request):
+    """Render table view.
+    """
+    if request.method == 'POST':
+        # Make dictionary of tablename : class object
+        # This matches what the user selects in the drop down to the python obj.
+        tables_of_interest = {}
+        for item in di.__dict__.keys():
+            table = getattr(di, item)
+            if hasattr(table, '__tablename__'):
+                tables_of_interest[table.__tablename__] = table
+
+        session, base, engine, meta = load_connection(get_config()['connection_string'])
+        tablename = request.POST['db_table_select']
+        table = tables_of_interest[tablename]  # Select table object
+        result = session.query(table)
+        result_dict = [row.__dict__ for row in result.all()]  # Turn query result into list of dicts
+        column_names = table.__table__.columns.keys()
+
+        # Build list of column data based on column name.
+        data = []
+        for column in column_names:
+            column_data = list(map(itemgetter(column), result_dict))
+            data.append(column_data)
+
+        # Build table.
+        tbl = Table(data, names=column_names)
```

Comment:
Could you rename this to something a bit more explicit?  Maybe `table_to_display` or something like that (since `table` is already used above).